### PR TITLE
Added description to README.md

### PR DESCRIPTION
**Pull request by shindesimantini6**
Please check if my changes are OK.

### CODE REVIEW

**Review by gsaci — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# REDI_Data_Analytics_Git_class
```
Comment (MiraySardogan) [inline README.md:1]: OK

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+# REDI_Data_Analytics_Git_class
```
Comment (abatanero) [inline README.md:1]: We could add a new line between title and description

**Review by saamehg — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,2 @@
...
+This is a test repository for the Data ANalytics class.
```
Comment (FathimaHaseeb) [inline README.md:2]: good.

File: README.md (hunk 2 above)
Comment (abatanero) [reply README.md:1]: Otherwise looks great! <3

**Comment by NataMurat** [comment]
!

**Review by abatanero — CHANGES_REQUESTED** [review]

**Review by NataMurat — COMMENTED** [review]
cool

**Review by nms289 — COMMENTED** [review]
Great!

**Pull request merged**